web/handlers/admin/invites.go

```diff
@@ -0,0 +1,73 @@
+package admin
+
+import (
+	"fmt"
+	"net/http"
+
+	"go.mindeco.de/http/render"
+
+	"github.com/gorilla/csrf"
+	"github.com/ssb-ngi-pointer/go-ssb-room/admindb"
+	"github.com/ssb-ngi-pointer/go-ssb-room/web/user"
+)
+
+type invitesH struct {
```

Comment:
"invitesH"? what does the H stand for?

Reply:
h as in handler. I guess a bit verbosity and spelling that out wouldn't hurt.

This follows from https://github.com/ssb-ngi-pointer/go-ssb-room/pull/55#discussion_r587330607 and the idea is to have all the receivers of the handlers can be `h`, too.

Reply:
ah! yes i wouldn't mind Handler instead :)

verbosity inversely proportional to how many times you have to type it out makes sense to me :3